### DISCUSSION TITLE
Great Module. Just want to know something.

### DISCUSSION THREAD

**Discussion by nolimitech** [TomSchimansky/CustomTkinter#681] [Ideas]
Hello.
Great job. Beautiful widgets.

I just want to know, why the methods doesn't have the same name as the real Tkinter?
It would be so nice, to change the name of the module and keep the code intact. Instead of rewriting everything based on your methods (Everything starting with CTk).

```python3
import tkinter as tk
# We should be able to do:
import customtkinter as tk

root = tk.Tk()

# We should be able to take the same methods as Tkinter.
label = tk.Label(root, text="Label")
# Instead of doing this... with CTk based everywhere.
label = tk.CTkLabel(master=root, text="CTkLabel")

root.mainloop()
```

As I said, everything is fine, except this. We need to create a new classes (Or Modules) just to convert the old Tkinter scripts. And the worst is... everything is the same, same widgets, same parameters, .... but not the names of the methods.....

I just want to know, why, why, why it's not the same? Is there a reason? haha

Thanks you ;)

**Comment by TomSchimansky** [comment]
They are named differently because they are not the same as tkinter widgets. They're something completely different. CustomTkinter is not just a theme applied to the original tkinter widgets, the widgets are created from scratch and rendered on a canvas. Also the arguments are not the same. It would be very bad coding style to name the widgets the same as the tkinter ones and it would be confusing. Also you could not use them in combination with the original tkinter widgets, although this is not the main reason.

**Comment by nolimitech** [comment]
Thank you, for the answer ;)
